### PR TITLE
Updated XCode task - teamId does not require signingOption auto anymore

### PR DESCRIPTION
**Pull request by leonidumanskiy**
Fixing #10312

### CODE REVIEW

File: Tasks/XcodeV5/task.json [hunk 1]
```diff
@@ -239,8 +239,7 @@
             "defaultValue": "",
```
Comment (madhurig) [inline Tasks/XcodeV5/task.json]: Please update the minor version of the task to "152"

File: Tasks/XcodeV5/task.json [hunk 2]
```diff
@@ -239,8 +239,7 @@
...
             "required": false,
             "helpMarkDown": "(Optional, unless you are a member of multiple development teams.) Specify the 10-character development team ID.",
-            "groupName": "sign",
-            "visibleRule": "signingOption = auto"
```
Comment (hashtagchris) [inline Tasks/XcodeV5/task.json]: This seems invalid in combination with "nosign" (`CODE_SIGNING_ALLOWED=NO`). I think the visibleRule should be `"visibleRule": "signingOption != nosign"`

**Comment by leantk** [comment]
This PR is stale because it has been open for a year with no activity. Remove the stale label or comment on the PR otherwise this will be closed in 5 days

**Comment by leonidumanskiy** [comment]
Afaik this is still a problem. I'll work on addressing the review comments

**Comment by MaxGuernseyIII** [comment]
Why did this get closed? 

The rule was "comment in five days or remove the stale tag" and the author commented within five days.

**Comment by leantk** [comment]
@MaxGuernseyIII thanks for the heads up. The Action I'm using shouldn't have closed it. I will investigate and see why.

**Pull request closed without merging**